docs/Build-A-Portable-Bottle.md

```diff
@@ -0,0 +1,71 @@
+# Building A Portabe Bottle for Linux
+
+## Introduction
+
+The bottles for `binutils`, `patchelf` and `zlib` are special. Because they are installed before `glibc`, they must be able to run on host systems using an older version of `glibc`. Linuxbrew supports host systems with `glibc` 2.13 and newer. We use Debian 7 (wheezy) with glibc 2.13 to build portable bottles.
+
+## Here are the instructions to build and upload the bottle
+
+```sh
+docker pull debian/eol:wheezy
+docker run -it --name=linuxbrew-wheezy debian/eol:wheezy
+```
+
+# Once in the running container, install the needed build dependencies
+```sh
+apt-get update
+apt-get install -y bison flex texinfo gcc g++ make curl sudo git-core
+```
+
+# Install brew
+```sh
+useradd -m -s /bin/bash linuxbrew
+echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+su -l linuxbrew
+
+export PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH
+export HOMEBREW_CURL_PATH=/usr/bin/curl
+export HOMEBREW_NO_AUTO_UPDATE=1
+export HOMEBREW_NO_ANALYTICS=1
+export HOMEBREW_DEVELOPER=1
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
+```
+
+# Setup git config to be able to pull your PR
+```sh
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+
+brew pull <PR_NUMBER>
+```
+
+# Prevent brew to install brewed curl and git
+
+To force brew to use the system curl and git, we need to modify brew.sh
```

Comment:
This seems like a bad idea to rely on. I'd suggest adding undocumented environment variables (that require `HOMEBREW_DEVELOPER` to be set, too) to override this.

Reply:
I suggest that if `HOMEBREW_DEVELOPER=1` and `HOMEBREW_CURL_PATH=1` are set, then that should defeat the version check.

Reply:
Could be even more specific still and say if they `_PATH` is set to `/usr/bin/...` and print a warning.

Reply:
Sure, fine by me.